firefox-ios/Client/Frontend/Settings/SearchSettingsTableViewController.swift

```diff
@@ -116,6 +113,30 @@ class SearchSettingsTableViewController: ThemedTableViewController, FeatureFlagg
     override func viewDidDisappear(_ animated: Bool) {
         super.viewDidDisappear(animated)
         setEditing(false, animated: false)
+        unsubscribeFromRedux()
+    }
+
+    // MARK: - Redux
+
+    func subscribeToRedux() {
+        store.dispatch(SearchSettingsAction.searchSettingsDidLoad)
+        store.subscribe(self, transform: {
+            $0.select(SearchSettingsState.init)
+        })
+    }
+
+    func unsubscribeFromRedux() {
+        store.unsubscribe(self)
+    }
+
+    func newState(state: SearchSettingsState) {
+        searchSettingsState = state
+        self.navigationItem.title = .Settings.Search.Title
```

Comment:
given that this isn't changing, should it be in a state?

cc @OrlaM

Reply:
Only dynamic parts of the view should be updated here. Everything static can be set up somewhere else (init, view did load etc)